app/controllers/catalog_controller.rb

```diff
@@ -41,6 +42,31 @@ def update_user_per_cat_page
     end
   end
 
+  def submit_cart_updates_form # rubocop:disable MethodLength, AbcSize
+    flash.clear
+    begin
+      # set the start and end date to updated values
+      # converts the dates into the expected format
+      start_date = Date.strptime(params[:form][:start_date], '%m/%d/%Y')
```

Comment:
I hear that, but ideally we should just be consistent between the two forms, one way or another. Since we've already got working JS code to handle the datepickers and extracting the data for the cart updates, we can basically embed the same form (structurally) on this page, which will allow us to use the `update_cart` method unmodified.